init/SitePackage.lua

```diff
@@ -0,0 +1,29 @@
+require("strict")
+local hook = require("Hook")
+
+-- from https://stackoverflow.com/a/40195356
+--- Check if a file or directory exists in this path
+function exists(file)
+	local ok, err, code = os.rename(file, file)
+	if not ok then
+		if code == 13 then
+			-- Permission denied, but it exists
+			return true
+		end
+	end
+	return ok, err
+end
+
+local function visible_hook(modT)
+	local frameStk  = require("FrameStk"):singleton()
+	local mt        = frameStk:mt()
+	local cudaDir = exists('/cvmfs/pilot.eessi-hpc.org/host_injections/nvidia/modules/')
```

Comment:
Oh, now I see how using `EESSI_SOFTWARE_PATH` might be hard for this. Can we use `os.getenv` here?

Reply:
We also probably also need an ENVVAR setting to force display of these modules